frontend/controllers/displayProductDetails.js

```diff
@@ -0,0 +1,63 @@
+document.addEventListener('DOMContentLoaded', onHtmlLoaded);
+
+function onHtmlLoaded() {
+
+const id =  window.localStorage.getItem("productId");

```

Comment:
Good job with the event listener, the lines above being part of the onHtmlLoaded function please add the indentations to them.